src/modules/private.ts

```diff
@@ -151,6 +151,13 @@ export default class Private {
 
   // ============ Requests ============
 
+  async getGeneric(endpoint: string, params: {}): Promise<Data> {
```

Comment:
name is just a bit weird, can you make this `get` and make the private function `_get`